resources/authgear/templates/en/translation.json

```diff
@@ -246,5 +249,8 @@
   "show-password-label": "Show password",
   "hide-password-label": "Hide password",
 
+  "return-page-title": "Return to where you were",
```

Comment:
I think the wording of this page can use some amendment, but that's good enough for now.